index.html

```diff
@@ -0,0 +1,166 @@
+                        <img src="IMAGES/practice_instruments.png" alt="real-life-experience" class="p-image">
+                        <h5 class="p-title">Real Life Experince</h5>
+                    </div>
+                    <div class="p-description">
+                        <p class="p-more">
+                            In the last three modules pf the program, students start to participate in real life activities depending on their fields.
+                        </p>
+                    </div>
+                </div>
+                <!-- PROGRAM 5 -->
+                <div class="program contests">
+                    <div class="program-head">
+                        <img src="IMAGES/competition.png" alt="contests" class="p-image">
+                        <h5 class="p-title">Contests</h5>
+                    </div>
+                    <div class="p-description">
+                        <p class="p-more">
+                            At the end of every block(3 months), We hold a music contest where students present their creations to the public.
+                        </p>
+                    </div>
+                </div>
+                <!-- PROGRAM 6 -->
+                <div class="program grad-prom">
+                    <div class="program-head">
+                        <img src="IMAGES/practice_instruments.png" alt="Promotion" class="p-image">
+                        <h5 class="p-title">Promotion</h5>
+                    </div>
+                    <div class="p-description">
+                        <p class="p-more">
+                            After students finish their one year program, they graduate. They are also promoted in the public through Darko Music.
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="see-more">SEE MORE PROGRAMS</div>
+            <button type="button" class="apply"><strong class="join">Apply Now</strong></button>
+        </section>
+        <!-- SPEAKERS -->
+        <section class="featured-speakers" id="speakers">
+            <div class="title"><h2 class="fp">Featured Speakers</h2></div>
+            <div class="underline"></div>
+            <div class="speakers">
+                <div class="left-side">
+                </div>
+                <div class="right-side"> 
+                </div>
+            </div>    
+            <button type="button" class="more-sp">MORE<img src="IMAGES/arrow-down.png" class="arrow-down" alt=""></button>
+        </section>
+        <section class="partners">
+            <div class="title-p"><h2 class="title-par">Partners</h2></div>
+            <div class="underline partn"></div>
+        <div class="partners-names">
+            <div class="par wtb">WE THE BEST</div>
+            <div class="par dotcom"><p class="dot">DotCom</p></div>
+            <div class="par young"><p class="ym">YOUNG MONEY</p></div>
+        </div>
```

Comment:
- [ ] Kindly wrap the list of the partners name inside of ul and every item should be a li.
 - [OPTIONAL] I suggest decreasing the size of the text because it will look better.